src/main/resources/l10n/JabRef_de.properties

```diff
@@ -2267,3 +2271,7 @@ Remove\ redundant\ spaces=Redundante Leerzeichen entfernen
 Replaces\ consecutive\ spaces\ with\ a\ single\ space\ in\ the\ field\ content.=Ersetzt aufeinanderfolgende Leerzeichen durch einen einzigen Leerzeichen im Feldinhalt.
 Remove\ digits=Ziffern entfernen
 Removes\ digits.=Ziffern entfernen.
+
+Citation\ relations=Zitierungsrelationen
```

Comment:
Just FYI as translations are handled by crowdin.com all non English modifications will be overwritten by crowdin,